sql/moz-fx-data-shared-prod/telemetry_derived/newtab_clients_daily_v1/query.sql

```diff
@@ -0,0 +1,168 @@
+WITH cte AS (
+  SELECT
+    client_id,
+    submission_date,
+    legacy_telemetry_client_id,
+    newtab_visit_id,
+    normalized_os,
+    normalized_os_version,
+    country_code,
+    locale,
+    channel,
+    browser_version,
+    browser_name,
+    default_search_engine,
+    default_private_search_engine,
+    pocket_is_signed_in,
+    pocket_enabled,
+    pocket_sponsored_stories_enabled,
+    topsites_enabled,
+    newtab_homepage_category,
+    newtab_newtab_category,
+    topsites_rows,
+    experiments,
+    had_non_impression_engagement,
+    had_non_search_engagement,
+    is_new_profile,
+    activity_segment,
+    search_interactions,
+    topsite_tile_interactions,
+    pocket_interactions
+  FROM
+    `moz-fx-data-shared-prod.telemetry_derived.newtab_visits_v1`
+  WHERE
+    submission_date = @submission_date
+),
+visits_data AS (
+  SELECT
+    client_id,
+    submission_date,
+    ANY_VALUE(legacy_telemetry_client_id) AS legacy_telemetry_client_id,
+    COUNT(DISTINCT newtab_visit_id) AS newtab_visit_count,
```

Comment:
This could probably also be a `COUNT`, IIRC `newtab_visits_v1` is one row per `newtab_visit_id`

Reply:
Ah yup, changed that